ckanext/sitesearch/interfaces.py

```diff
@@ -0,0 +1,93 @@
+from ckan.plugins.interfaces import Interface
+
+
+class ISiteSearch(Interface):
+    """
+    Hooks into the search actions to implement custom logic.
+
+    This methods will be called before/after the search is performed to allow
+    users to modify the search parameters or the search results.
+
+    For the case of before_* methods, extensions will receive a dictionary with
+    the query parameters, and should return a modified (or not) version of it.
+
+    For the case of after_* methods (except site_search), extensions will
+    receive the search results, as well as the search parameters, and should
+    return a modified (or not) object with the same structure::
+
+        {'count': '', 'results': '', 'facets': ''}
```

Comment:
`facets` is deprecated. We should use `search_facets`

http://docs.ckan.org/en/latest/api/index.html#ckan.logic.action.get.package_search

Can you do a quick replace on all docstrings?